packages/react-devtools-shared/src/hook.js

```diff
@@ -334,7 +334,7 @@ export function installHook(target: any): DevToolsHook | null {
     // For React Native, we intentionally patch early (during injection).
     // This provides React Native developers with components stacks even if they don't run DevTools.
     //
-    // This won't work for DOM though, since this entire file is eval'ed and inserted as a script tag.
+    // This won't work for DOM though, since this entire file is evaded and inserted as a script tag.
```

Comment:
The word eval'ed means evaluated and it's not same as evaded. Other corrections are not typos actually.

Reply:
may be evaled is evaluated but evaling has no meaning and evading is the closest to the actual meaning fitt to the sentence

Reply:
> Not needed.

Never reply "not needed" unless you are a part of the maintainers, let them comment what they find incorrect the typos i have tried to fix first of all are typos ,evaling does not have any meaning, envified has no meaning , if you know it is not needed then sugges whats needed and that what maintainer of the project will do in a review